common/docs/SIE_ACS_Verification.md

```diff
@@ -45,7 +45,17 @@ cd ..
 rm -rf swtpm/ libtpms/
 ```
 
-## Building UEFI Firmware
+## Running SIE ACS on QEMU with UEFI firmware
+
+### Build QEMU model
+Follow build instructions from https://www.qemu.org/download/#source
+
+Note: During configure stage, enable slirp library build by appending ./configure with --enable-slirp
```

Comment:
it is a networking support library. Since run command as netdev it is required.

Reply:
Added a note.